includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -2,20 +2,64 @@
 
 class DesignSystemApiController extends WikiaApiController {
 	public function getFooter() {
-		$wikiId = $this->request->getInt( 'wikiId' );
-		$lang = $this->request->getVal( 'lang' );
+		$params = $this->getRequestParameters();
+
+		$footerModel = new DesignSystemGlobalFooterModel( $params[ 'wikiId' ], $params[ 'lang' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikiId" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)